### ISSUE TITLE
Change location of notes files

### ISSUE THREAD

**Issue by derkommandant** [standardnotes/desktop#167]
There is not an option to change the location where the note files are dropped (using MacOS High Sierra). By default they are all put in the logged in user document folders. Is there a way to change the location?

**Comment by mobitar** [comment]
Not yet, but this is something on my mind.

**Comment by seascape** [comment]
Please give us the option to store all user data in one place, such as the location of the executable. 

As a Windows user I prefer portable apps whenever possible. Having important data strewn around the C:\Users folders makes backup and migration much more of a hassle.

**Comment by mobitar** [comment]
Indeed, definitely something that would be useful. Have yet to determine if this is possible with Electron/electron-builder.

**Comment by seascape** [comment]
This suggests that at least app install path can be user-specified.

https://stackoverflow.com/questions/45661129/how-to-set-a-custom-path-for-electron-app-installer

**Comment by leafcutterant** [comment]
I too would be very grateful for a portable version on Windows (meaning that the installer just "extracts" the program files to a user-defined folder and the app doesn't rely on anything outside of that folder, even the user data resides in there).

**Comment by vensko** [comment]
It's an Electron app, so standard Chromium swtiches still work:
`"Standard Notes.exe"  "--user-data-dir=YOUR_PATH\Standard Notes"`

**Comment by JaspalSuri** [comment]
Hi @vensko, thank you for sharing that. 🙂
